doc/doxygen/CMakeLists.txt

```
@@ -175,7 +175,8 @@ add_custom_command(
          ${CMAKE_CURRENT_BINARY_DIR}/footer.html
   COMMAND ${CMAKE_COMMAND} -E touch header.html
   COMMAND ${CMAKE_COMMAND} -E touch footer.html
-  COMMAND ${DOXYGEN_EXECUTABLE} -w html header.html footer.html style.css options.dox
+  COMMAND ${DOXYGEN_EXECUTABLE} -w html header.html footer.html
+                                   doxygen-awesome.css custom.css options.dox
```

I think the problem with this PR is in this line. As far as I understand the doxygen documentation the .css files given here is one (and exactly one!) *output* file, not a list of css input files. In other words the fact that this file was called `style.css` before was not related to the `style.css` input file in doc/doxygen. I tested this locally for me, and with your change I get the same error message as the CI, but when I do the following everything works and builds with the new design:

```suggestion
                                   style.css options.dox
```
@pcafrica Do you want to give this a try and see what the testers say?